lib/DDGC/Web/Controller/Ideas.pm

```diff
@@ -352,6 +363,20 @@ sub vote_view :Chained('vote') :PathPart('') :Args(0) {
 	};
 	$c->forward( $c->view('JSON') );
 }
+sub format_meta_id {
+    my( $id ) = @_;
+
+    # meta_id must be lowercase and without weird chars
+    $id = lc $id;
+    $id =~ s/[^a-z0-9]+/_/g;
+    $id =~ s/^[^a-zA-Z]+//;
+    $id =~ s/_$//;
```

Comment:
You can't have multiple repeated ones, because line 371 already turns multiple weird chars (underscore included) into a single underscore.